### PR TITLE
Creating the Biblio contributor UI

### PR DESCRIPTION
**Pull request by RoySegall**
#22

add
delte
edit
view

### CODE REVIEW

File: modules/biblio_contributor_ui/biblio_contributor_ui.module [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+  $items['admin/biblio-contributor/add'] = array(
+    'title' => 'Add biblio contributor',
+    'access arguments' => array('create biblio contributor entry'),
+    'page callback' => array('drupal_get_form'),
+    'page arguments' => array('biblio_contributor_ui_create_form'),
+    'type' => MENU_LOCAL_ACTION,
+  );
+
+  $items['biblio-contributor/%biblio_contributor'] = array(
+    'title' => 'View',
```
Comment (amitaibu) [inline modules/biblio_contributor_ui/biblio_contributor_ui.module]: This should be a title callback -- look at how Biblio UI now deals with it.

File: modules/biblio_contributor_ui/biblio_contributor_ui.module [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+    ),
+    'update biblio contributor entry' => array(
+      'title' => t('Update biblio contributor entry'),
+      'description' => t('Grant to the user the permission to update a biblio contributor via the UI'),
+    ),
+    'delete biblio contributor entry' => array(
+      'title' => t('Delete biblio contributor entry'),
+      'description' => t('Grant to the user the permission to delete a biblio contributor via the UI'),
+    ),
+    'delete own biblio contributor entry' => array(
```
Comment (amitaibu) [inline modules/biblio_contributor_ui/biblio_contributor_ui.module]: Remove.

File: modules/biblio_contributor_ui/biblio_contributor_ui.module [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+      'title' => t('Delete own biblio contributor entry'),
+      'description' => t('Grant to the user the permission to delete his own biblio contributor entry via the UI'),
+    ),
+    'view biblio contributor entry' => array(
```
Comment (amitaibu) [inline modules/biblio_contributor_ui/biblio_contributor_ui.module]: Remove

File: modules/biblio_contributor_ui/biblio_contributor_ui.module [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+
+  return $items;
+}
+
+/**
+ * Implements hook_permission().
+ */
+function biblio_contributor_ui_permission() {
+  return array(
+    'manage biblio contributor' => array(
```
Comment (amitaibu) [inline modules/biblio_contributor_ui/biblio_contributor_ui.module]: Remove.

File: modules/biblio_contributor_ui/biblio_contributor_ui.module [hunk 5]
```diff
@@ -0,0 +1,291 @@
...
+  );
+}
+
+/**
+ * Manage biblio contributor UI.
+ */
+function biblio_contributor_ui_create_form($form, &$form_state, BiblioContributor $contributor = NULL) {
+  global $user;
+
+  $account = user_load($user->uid);
```
Comment (amitaibu) [inline modules/biblio_contributor_ui/biblio_contributor_ui.module]: Can probably be removed.

File: modules/biblio_contributor_ui/biblio_contributor_ui.module (hunk 3 above)
Comment (RoySegall) [inline modules/biblio_contributor_ui/biblio_contributor_ui.module]: If removing, which permission should handle the access to view the author? Or should we remove the view of the contributor entry?

**Comment by amitaibu** [comment]
Merged.

**Pull request closed without merging**